projects/plugins/jetpack/extensions/plugins/launchpad-save-modal/index.js

```diff
@@ -25,6 +27,7 @@ export const settings = {
 		}, [ isSaving, prevIsSaving ] );
 
 		return (
+			currentSite.launchpadScreenOption === 'full' &&
```

Comment:
neat!

I imagine we will also need to check the specific flow, since the current copy is very specific to LIB (and i imagine we could follow up to add other variations for other flows). I think we were using a `site_intent` option or something like that?

Reply:
Second all of Addison's thoughts. I also think the CTA "Launch Site" is a little misleading, since all it does is redirect to Launchpad.

Maybe we could come up with a more context appropriate name like "Next Steps"

Reply:
> Second all of Addison's thoughts. I also think the CTA "Launch Site" is a little misleading, since all it does is redirect to Launchpad.

+1 on this.

I got confused for a moment when I was initially testing because I thought it was going to immediately launch my site based on the text instead of sending me back to the LaunchPad screen.

> Maybe we could come up with a more context appropriate name like "Next Steps"

I think "Next Steps" should be sufficient here since its consistent with the calypso/wp-admin sidebar launchpad notice we just added and is not as confusing.